### ISSUE TITLE
Inline WHILE triggers SpaceBeforeSemicolon incorrectly

### ISSUE THREAD

**Issue by sybrew** [squizlabs/PHP_CodeSniffer#2562]
Test code 1:
```php
$level = ob_get_level();
while ( $level-- ) ob_end_clean();
```

Error triggered 1:
```
Expected 0 spaces before semicolon; 1 found
Squiz.ControlStructures.ControlSignature.SpaceBeforeSemicolon
```

--------

Test code 2:

```php
$level = ob_get_level();
while ( $level-- )
	ob_end_clean();
```

Error triggered 2:
```
Expected 0 spaces before semicolon; newline found
Squiz.ControlStructures.ControlSignature.SpaceBeforeSemicolon
```

--------

**Additional notes:**
An inline for-loop doesn't trigger such an error.

**Comment by jrfnl** [comment]
Clear report. Bug confirmed. PR #2563 should fix this.
